src/dymaptic.GeoBlazor.Core/Components/Widgets/LayerList/BaseListItem.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Microsoft.AspNetCore.Components;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace dymaptic.GeoBlazor.Core.Components.Widgets.LayerList
+{
+    /// <summary>
+    ///     The ListItem class represents one of the operationalItems in the LayerListViewModel. In the LayerList widget UI, the list item represents a layer displayed in the view. It provides access to the associated layer's properties, allows the developer to configure actions related to the layer, and allows the developer to add content to the item related to the layer.
+    ///     <a target="_blank" href="https://developers.arcgis.com/javascript/latest/api-reference/esri-widgets-BasemapLayerList.html">ArcGIS JS API</a>
+    /// </summary>
+    public class BaseListItem
```

Comment:
This class should not exist. It is duplicating `ListItem` and it is not used.

Reply:
baselistitem is removed